RubeesEat/wwwroot/css/site.css

```diff
@@ -113,6 +110,193 @@ nav > ul > li:hover {
+    display: inline-block;
+    margin-left: .5rem;
+    font-weight: normal;
+}
+
+/* Buttons */
+.button,
+button,
+input[type="submit"],
+input[type="reset"],
+input[type="button"] {
+    display: inline-block;
+    height: 38px;
+    padding: 0 30px;
+    color: var(--primary-color-darker);
+    text-align: center;
+    font-size: 11px;
+    font-weight: 600;
+    line-height: 38px;
+    letter-spacing: .1rem;
+    text-transform: uppercase;
+    text-decoration: none;
+    white-space: nowrap;
+    background-color: transparent;
+    border-radius: 4px;
+    border: 1px solid var(--primary-color-darker);
+    cursor: pointer;
+}
+
+.button:hover,
+button:hover,
+input[type="submit"]:hover,
+input[type="reset"]:hover,
+input[type="button"]:hover
+{
+    color: var(--primary-color);
+    border-color: var(--primary-color);
+    outline: 0; 
+}
+
+.button:focus,
+button:focus,
+input[type="submit"]:focus,
+input[type="reset"]:focus,
+input[type="button"]:focus
+{
+    color: var(--primary-color-lighter);
+    border-color: var(--primary-color-lighter);
+    outline: 0; 
+}
+
+.button:disabled,
+button:disabled,
+input[type="submit"]:disabled,
+input[type="reset"]:disabled,
+input[type="button"]:disabled
+{
+    color:gray;
+    border-color: gray;
+    outline: 0;
+}
+
+.button.button-primary,
+button.button-primary,
+input[type="submit"].button-primary,
+input[type="reset"].button-primary,
+input[type="button"].button-primary {
+    color: var(--secondary-color);
+    background-color: var(--primary-color-darker);
+    border-color: var(--secondary-color); 
+}
+
+.button.button-primary:hover,
+button.button-primary:hover,
+input[type="submit"].button-primary:hover,
+input[type="reset"].button-primary:hover,
+input[type="button"].button-primary:hover {
+    color: var(--secondary-color);
+    background-color: var(--primary-color);
+    border-color: var(--secondary-color); 
+}
+
+a.button.button.button-primary:focus,
+button.button-primary:focus,
+input[type="submit"].button-primary:focus,
+input[type="reset"].button-primary:focus,
+input[type="button"].button-primary:focus {
+    color: var(--secondary-color);
+    background-color: var(--primary-color-lighter);
+    border-color: var(--secondary-color); 
+}
+
+.entryLines {
+    list-style: none;
+    padding: 0;
+    margin: 10px;
+}
+
+.entryLines li {
+    margin: 10px 10%;
+    padding: 10px;
+    background-color: white;
+    border-radius: 5px;
+    box-shadow: 10px 10px 10px 10px rgba(0, 0, 0, 0.1);
+    transition: all 0.3s ease;
+}
+
+.entryLines {
+    --positive-background-color: #FFC1C1;
+    --negative-background-color: #E7FFE7;
+}
+
+.entryLines li.negative-amount {
+    background-color: var(--negative-background-color);
+}
+.entryLines li.positive-amount {
+    background-color: var(--positive-background-color);
+}
+
+.amount.negative-amount {
+    color: red;
+}
+
+.amount.positive-amount {
+    color: green;
+}
+
+.entryLines li:hover {
+    transform: scale(1.02);
+}
```

Comment:
I think this hover effect should only apply to the entrylines in home and not on billdetails too since this is not clickable